### PR TITLE
CDAP-12239 Document steps to perform when upgrading the replicated clusters.

### PR DESCRIPTION
**Pull request by sagarkapare**

### CODE REVIEW

File: cdap-docs/admin-manual/source/installation/replication.rst [hunk 1]
```diff
@@ -255,3 +255,56 @@ To manually failover from the master to a slave cluster, follow these steps:
...
+
+    $ sudo apt-get install --only-upgrade '^cdap.*'
+
+3. Generate the coprocessor jar on the slave cluster corresponding to the newly downloaded CDAP version::
+
+       $ sudo -u <cdap-user> cdap setup coprocessors
+
+   The coprocessor jar will be stored on the HDFS and path to the jar will be printed on console.
+
+4. Copy the coprocessor jar at the same HDFS location on the master cluster.
```
Comment (albertshau) [inline cdap-docs/admin-manual/source/installation/replication.rst]: 'jar at the' -> 'jar to the'

File: cdap-docs/admin-manual/source/installation/replication.rst [hunk 2]
```diff
@@ -255,3 +255,56 @@ To manually failover from the master to a slave cluster, follow these steps:
...
+7. Run the upgrade tool on the slave cluster.::
+
+     $ sudo -u <cdap-user> /opt/cdap/master/bin/cdap run co.cask.cdap.data.tools.UpgradeTool upgrade
+
+   Since replication is enabled and HBaseDDLExecutor is in place, HBase tables on the master cluster will also get
+   upgraded.
+
+8. New version of the CDAP can be started on the slave cluster now.
+
+9. Download and install the new CDAP packages on the master cluster using the steps mentioned above for slave. This is to keep master ready when there is a failover from slave. Note that no need to run the upgrade tool on the master since HBase tables on master are already upgraded when the upgrade tool was executed on the slave.
```
Comment (albertshau) [inline cdap-docs/admin-manual/source/installation/replication.rst]: 'keep master ready' -> 'keep the master ready'

Comment (albertshau) [reply cdap-docs/admin-manual/source/installation/replication.rst]: 'Note that no need' -> 'Note that there is no need'

Comment (albertshau) [reply cdap-docs/admin-manual/source/installation/replication.rst]: 'are already upgraded' -> 'were already upgraded'

**Review by albertshau — APPROVED** [review]
just a few grammatical corrections, otherwise lgtm. Fix and merge.

**Comment by sagarkapare** [comment]
Addressed comments. Squashing and merging.

**Pull request merged**